src/components/InstructionsPopup.jsx

```diff
@@ -1,10 +1,30 @@
 import React from 'react';
+import {IoPlayOutline, IoPauseOutline, IoArrowForwardOutline, IoArrowBackOutline, IoArrowDownOutline, IoRefreshOutline, IoClose} from 'react-icons/io5';
+import StyledInstructionsPopup from './styles/StyledInstructionsPopup';
 
-const InstructionsPopup = () => {
+const InstructionsPopup = ({onPopupHandle}) => {
 	return (
-		<div>
-			
-		</div>
+		<StyledInstructionsPopup>
+			<header>
+				<h2>How to play</h2>
+				<IoClose onClick={onPopupHandle}/>
+			</header>
+			<p>Bring down blocks (tetrominos) from the top of the screen. You can move the blocks left and right and also rotate them clockwise.</p>
+			<p>Your objective is to get tetrominos to fill all the empty cells in horizontal lines; whenever you do this, the blocks vanish and you earn points.</p>
+			<p>More points == higher levels == higher speed == greater difficulty.</p>
+			<div>
+				<span><IoPlayOutline/><IoPauseOutline/></span>
+				<span>start game / pause</span>
```

Comment:
should we use the icons to explain even we dont use them in the interface?

Reply:
Let's replace them with button representations in this case. I'm thinking about adding kind of a media query in react — to display either icon or button text depending on the screen and what's on it anyway. Will add a new to-do.